.gitignore

```diff
@@ -1,3 +1,4 @@
 /build
 /**/node_modules/.cache
-.idea/
\ No newline at end of file
+e2e_*
```

Comment:
Where is this used?

Reply:
https://github.com/efficientgo/e2e creates tmp local e2e_<id> dir for every test run.

Reply:
Hm, why local? Why not in the OS tmp dir?

Reply:
That's the improvement on https://github.com/efficientgo/e2e if we want that. Was easier with perms and sharing with docker, but maybe just laziness (: